elasticizer/__main__.py

```diff
@@ -4,33 +4,41 @@
 import random
 from elasticizer import Load
 import collections
+import configargparse
+
 
 def buildArgParser():
```

Comment:
Just a bit of QA, the function calls in this method probably do not pass pylint or PEP8 compliance